util/msys2_install.sh

```diff
@@ -8,25 +8,26 @@ installflip=false
 util_dir=$(dirname "$0")
 
 echo "Installing dependencies needed for the installation (quazip)"
-pacman --needed -S base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-clang msys/git msys/p7zip msys/python3 msys/unzip
+pacman --needed --noconfirm -Sy base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-clang msys/git msys/p7zip msys/python3 msys/unzip
 
 source "$dir/win_shared_install.sh"
 
 function install_avr {
     rm -f -r "$avrtools"
-    wget "http://ww1.microchip.com/downloads/en/DeviceDoc/avr8-gnu-toolchain-3.6.1.1752-win32.any.x86.zip"
+    wget "https://blog.zakkemble.net/download/avr-gcc-8.3.0-x86-mingw.zip"
```

Comment:
Have we got in touch with this guy letting him know that his webhosting usage may skyrocket soon? :^)

Reply:
Did some sleuthing, he seems to be using Afterburst, which claims to be unmetered, so might be OK 😄

Reply:
If it does turn out to be an issue then I have an unmetered VPS we can mirror it on

Reply:
I mean, absolute worst case, we could dump it into the qmk.fm repo. It wouldn't be the first toolchain that is handled that way: 
https://github.com/qmk/qmk.fm/